python/cuml/preprocessing/model_selection.py

```diff
@@ -49,7 +52,7 @@ def train_test_split(
         of instances to be assigned to the training set. Defaults to 0.8
     shuffle : bool, optional
         Whether or not to shuffle inputs before splitting
-    seed : int, CuPy RandomState or NumPy RandomState optional
```

Comment:
Should we keep `seed` in there for the time being and add that it is deprecated in favor of `random_state`?